prospero-common/src/main/resources/org/wildfly/prospero/Messages.i18n_cs_CZ.properties

```diff
@@ -0,0 +1,2 @@
+dirMustBeDirectory=Daná cesta %s je soubor. Je třeba prázdný adresář. 
```

Comment:
Wrong PR?

Reply:
Yes, that's not supposed to be there. Updated.